src/rtp/srtp.rs

```diff
@@ -650,6 +660,19 @@ impl fmt::Debug for Derived {
     }
 }
 
+fn error_details(header: &RtpHeader, srtp_index: u64) -> String {
```

Comment:
nit: You can maybe avoid allocating here by returning a `std::fmt::Arguments` produced from `format_args!`

Reply:
```rust
fn error_details<'a>(header: &'a RtpHeader, srtp_index: u64) -> Arguments<'a> {
    format_args!(
        "SSRC: {} seq_no: {} pt: {} mid: {:?} rid: {:?} rid_repair: {:?} srtp_index: {}",
        header.ssrc,
        header.sequence_number,
        header.payload_type,
        header.ext_vals.mid,
        header.ext_vals.rid,
        header.ext_vals.rid_repair,
        srtp_index
    )
}```

this doesn't compile with:

```
cannot return value referencing function parameter `srtp_index`
returns a value referencing data owned by the current function
```

Do you see a way around that?

Reply:
Hm, I guess `format_args!` captures `srtp_index` by reference rather than by copy... I don't see a way around that no, not super important anyway :)